src/compiler/compile/render-dom/wrappers/Window.ts

```diff
@@ -92,29 +92,29 @@ export default class WindowWrapper extends Wrapper {
 
 				renderer.meta_bindings.add_block(deindent`
 					if (${condition}) {
-						window.scrollTo(${x || 'window.pageXOffset'}, ${y || 'window.pageYOffset'});
+						@_scrollTo(${x || '@_pageXOffset'}, ${y || '@pageYOffset'});
 					}
-					${x && `${x} = window.pageXOffset;`}
-					${y && `${y} = window.pageYOffset;`}
+					${x && `${x} = @_pageXOffset;`}
```

Comment:
I imagine these would need to be `@_window.page_Offset` as well then, for the same reason?

Reply:
Yeah, all offsets. I didn't see any other global that'd have the same problem, though. And I didn't find any other missing globals either!